index.bs

```diff
@@ -284,19 +292,29 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     it. Subsequently, only that [RP], as identified by its <a>RP ID</a>, is able to employ the <a>scoped credential</a> in
     <a>authentication</a> ceremonies, via the <a>getAssertion()</a> method. The [RP] uses its copy of the stored public key to
     verify the resultant <a>Authentication Assertion</a>.
+    
+: <dfn>Test of User Presence</dfn>
+: <dfn>TUP</dfn>
+:: A [=test of user presence=] is a simple form of [=authorization gesture=] and technical process where a user interacts with
+    an [=authenticator=] by (typically) simply touching it (other modalities may also exist), yielding a boolean result. Note
+    that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
+    recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
 : <dfn>User Consent</dfn>
 :: User consent means the user agrees with what they are being asked, i.e., it encompasses reading and understanding prompts.
-    <a>User verification</a> encompasses the means employed by the user to indicate consent.
+    An [=authorization gesture=] is the [=ceremony=] component employed by a user to indicate consent.
 
 : <dfn>User Verification</dfn>
-:: The process by which an <a>authenticator</a> <em>locally authorizes</em> the invocation of the
-    <a>authenticatorMakeCredential</a> and <a>authenticatorGetAssertion</a> operations, for example through a touch plus pin
-    code, a password, a gesture (e.g., presenting a fingerprint), or other modality. Note that invocation of said operations
-    implies use of key material managed by the authenticator.
+:: The technical process by which an <a>authenticator</a> <em>locally authorizes</em> the invocation of the
+    <a>authenticatorMakeCredential</a> and <a>authenticatorGetAssertion</a> operations. [=User verification=] may be instigated
+    through various [=authorization gesture=] modalities; for example, through a touch plus pin code, password entry, or
+    [=biometric recognition=] (e.g., presenting a fingerprint) [[ISOBiometricVocabulary]]. Note that invocation of the
+    <a>authenticatorMakeCredential</a> and <a>authenticatorGetAssertion</a> operations implies use of key material managed by
+    the authenticator. Note that for security, [=user verification=] and use of [=credential private keys=] must occur within a
+    single logical security boundary defining the [=authenticator=].
```

Comment:
Should we also add a sentence saying that a "fresh enough" user verification can also implicitly provide a TUP?